tests/test_harvest_all_fields.py

```diff
@@ -0,0 +1,108 @@
+"""Test that all fields are received when all the fields are selected."""
+import re
+from tap_tester import menagerie, connections, runner
+from base import BaseTapTest
+from harvest_api import set_up_class, tear_cown_class
+
+KNOWN_MISSING_FIELDS = {
+    "external_reference": {
+        "task_id",
+    },
+    "invoice_messages": {
+        "send_reminder_on",
+    },
+    "invoices": {
+        "paid_at",
+        "paid_date",
+        "period_end",
+        "closed_at",
+        "period_start",
+    },
+    "estimates": {
+        "accepted_at",
+        "declined_at",
+    },
+    "time_entries": {
+        "timer_started_at",
+    },
+    "projects": {
+        "over_budget_notification_date",
+    },
+}
+
+
+class HarvestAutomaticFields(BaseTapTest):
```

Comment:
I think this is an all_fields test, can we change the name here and in the `name()` method as well?

Reply:
Renamed test class and name() method to all_fields..